manifests/lge_d802.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+<remote name="enes" 
+            fetch="https://github.com/"/>
+<project path="device/lge/d802" name="LineageOS/android_device_lge_d802" remote="enes" revision="cm-14.1" />
+<project path="device/lge/g2-common" name="LineageOS/android_device_lge_g2-common" remote="enes" revision="cm-14.1" />
+<project path="kernel/lge/msm8974" name="LineageOS/android_kernel_lge_msm8974" remote="enes" revision="cm-14.1" />
```

Comment:
You would need to have a kernel with modified defconfig as per https://docs.halium.org/en/latest/porting/build-sources.html#modify-the-kernel-configuration